frontend/src/lang/es.ts

```diff
@@ -484,7 +484,7 @@ export const es: LanguageConfig = {
         },
       },
       dob: {
-        header: "Verificar fecha de nacimiento",
+        header: "Access your COVID-19 test result (translate me!)",
```

Comment:
@Rebecca-LM let's don't forget about this 😂

Reply:
Acceda al resultado de su prueba de COVID-19